feature/search/src/main/kotlin/uk/govuk/app/search/ui/SearchScreen.kt

```diff
@@ -122,6 +124,7 @@ private fun SearchScreen(
 @Composable
 fun ShowResults(searchResults: List<Result>) {
     val viewModel: SearchViewModel = hiltViewModel()
+    val scope = rememberCoroutineScope()
```

Comment:
This isn't required, the view model function can launch a view model scope block.

Reply:
Fixed up